bids/layout/tests/test_validation.py

```diff
@@ -111,40 +111,6 @@ def test_is_session_level_true(testvalidator):
         assert result
 
 
-# checks is_session_level() function false cases
```

Comment:
This should pass with the release of bidsschematools 0.11.0. Would you mind restoring it to verify? Then removing the tests can all be in #1081.